src/modules/aidoku/DeepLink.ts

```diff
@@ -0,0 +1,27 @@
+import { Manga } from "./Manga";
+import { Chapter } from "./Chapter";
+import { create_deeplink } from "./aidoku";
+
+/**
+ * Directly open mangas from aidoku://:sourceId/:mangaId URLs.
+ */
+export class DeepLink {
+    /**
+     * Creates a new DeepLink object.
+     * @param manga The manga of the deep link.
+     * @param chapter The chapter of the manga.
+     */
+    constructor(public manga: Manga | null, public chapter: Chapter | null) {
```

Comment:
Although the rust wrapper uses an optional value for the manga field, Aidoku actually requires a manga object regardless, so it shouldn't be optional (I need to change the rust wrapper).

```suggestion
    constructor(public manga: Manga, public chapter: Chapter | null) {
```